### PR TITLE
docs: address small nits

### PR DESCRIPTION
**Pull request by andrewrynhard**
Small fixes to configuration comments, and docs website.

Signed-off-by: Andrew Rynhard <andrew@rynhard.io>

### CODE REVIEW

**Review by smira — DISMISSED** [review]

File: website/content/docs/v0.7/Reference/configuration.md [hunk 1]
```diff
@@ -1499,10 +1499,11 @@ Defaults to `false`.
 </div>
 <div class="dt">
 
-Specifies time (ntp) servers to use for setting system time.
+Specifies time (NTP) servers to use for setting the system time.
 Defaults to `pool.ntp.org`
 
-> Note: This parameter only supports a single time server
+
```
Comment (Unix4ever) [inline website/content/docs/v0.7/Reference/configuration.md:1505]: This extra whitespace probably will be caught by markdown lint.

File: pkg/machinery/config/types/v1alpha1/v1alpha1_types.go [hunk 2]
```diff
@@ -882,16 +880,17 @@ type Env = map[string]string
 
 // MachineFile represents a file to write to disk.
 type MachineFile struct {
-	//   description: The contents of file.
+	//   description: The contents of the file.
 	FileContent string `yaml:"content"`
-	//   description: The file's permissions in octal.
+	//   description: The file's permissions in decimal.
```
Comment (Unix4ever) [inline pkg/machinery/config/types/v1alpha1/v1alpha1_types.go:885]: Just fixed this one to be marshalled as octal :)

**Review by Unix4ever — APPROVED** [review]
with couple tiny things

**Review by smira — APPROVED** [review]

File: website/content/docs/v0.7/Reference/configuration.md (hunk 1 above)
Comment (andrewrynhard) [reply website/content/docs/v0.7/Reference/configuration.md:1519]: It would, but I think the generated markdown files have an ignore clause.

**Pull request merged**